### PR TITLE
 Check off todo list from readme file

### PR DESCRIPTION
**Pull request by sangttruong**

### CODE REVIEW

**Review by bryanhanson — CHANGES_REQUESTED** [review]
1. Re: the copyright holder, please stick to the language in the template.  All the authors jointly hold the copyright.
2. For `NEWS.md` you should keep the Acknowledgments section and it should thank GSOC for supporting both you and Erick.

**Review by bryanhanson — APPROVED** [review]
Looks good!

**Review by bryanhanson — APPROVED** [review]
Looks good!

File: NEWS.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# SKELETON
+# hySpc.read.jdx 0.0.x 2021-xx-xx
 
 ## Acknowledgements
-The `hyperSpec` team gratefully acknowledges support from the Google Summer of Code program, which sponsored student Erick Oduniyi during summer 2020.
-Erick and the team carried out a significant overhaul of `hyperSpec` which led to this release.
+The `hyperSpec` team gratefully acknowledges support from the Google Summer of Code program, which sponsored student Erick Oduniyi during Summer 2020 and Sang Truong during summer 2021.
```
Comment (GegznaV) [inline NEWS.md]: Were Erick working on this pacakge? Should he be mentioned here? @bryanhanson

File: LICENSE [hunk 2]
```diff
@@ -1,2 +1,22 @@
-YEAR: 2020-
-COPYRIGHT HOLDER: the SKELETON authors - see file DESCRIPTION
\ No newline at end of file
+YEAR: 2021-
```
Comment (GegznaV) [inline LICENSE:1]: Is it formally "legal" to change the license to MIT? @cbeleites

File: NEWS.md (hunk 1 above)
Comment (bryanhanson) [reply NEWS.md]: No Erick on this one.  Good catch!

File: LICENSE (hunk 2 above)
Comment (bryanhanson) [reply LICENSE:1]: We are OK on this package, it is entirely new and no material from the legacy version is being used (at least to this point, there will be unit tests and the vig but I'm not sure we would worry about this from a license POV).

File: NEWS.md (hunk 1 above)
Comment (sangttruong) [reply NEWS.md]: I corrected it.

**Pull request merged**